MovieHubRx/Scenes/Favorite/FavoriteViewModel.swift

```diff
@@ -18,15 +18,55 @@ struct FavoriteViewModel {
 extension FavoriteViewModel: ViewModelType {
     
     struct Input {
-        
+        let loadTrigger: Driver<Void>
+        let selectTrigger: Driver<IndexPath>
+        let refreshTrigger: Driver<Void>
     }
     
     struct  Output {
-        
+        let movies: Driver<[Movie]>
+        let error: Driver<Error>
+        let indicator: Driver<Bool>
+        let isRefreshing: Driver<Bool>
     }
     
     func transform(_ input: Input, disposeBag: DisposeBag) -> Output {
-   
-        return Output()
+        
+        let indicator = ActivityIndicator()
```

Comment:
thêm 1 cái refreshIndicator nữa e
```suggestion
        let activityIndicator = ActivityIndicator()
        let refreshIndicator = ActivityIndicator()
```